app/domain/authentication/authn_jwt/configuration_jwt_generic_vendor.rb

```diff
@@ -2,14 +2,27 @@ module Authentication
   module AuthnJwt
     # Mock JWTConfiguration class to use it to develop other part in the jwt authenticator
     class ConfigurationJWTGenericVendor < ConfigurationInterface
+      GENERIC_JWT_CONSTRAINTS = Authentication::Constraints::MultipleConstraint.new(
+            Authentication::Constraints::NotEmptyConstraint.new
+        )
+
       def self.jwt_identity(authentication_parameters)
         id_provider = Authentication::AuthnJwt::IdentityProviderFactory.new(authentication_parameters).relevant_id_provider
```

Comment:
why does the factory have a constructor? it should have only one static function named `create` that should receive the `authentication_parameters` as a param

Reply:
Not relevant to the scope of this PR. Its related to the PR of get identity
https://github.com/cyberark/conjur/pull/2139

Anyway because the get identity objects has identity_availiable function that only if it returns true we want to return them. And i want to create them one